client/package.json

```diff
@@ -1,617 +1,622 @@
+			},
+			{
+				"key": "shift+alt+i",
+				"mac": "shift+ctrl+i",
+				"command": "extension.coq.display.toggle.implicitArguments",
+				"when": "editorTextFocus && editorLangId == coq"
+			},
+			{
+				"key": "shift+alt+i",
+				"mac": "shift+ctrl+i",
+				"command": "extension.coq.display.toggle.implicitArguments",
+				"when": "resourceScheme==coq-view"
+			},
+			{
+				"key": "shift+alt+c",
+				"mac": "shift+ctrl+c",
+				"command": "extension.coq.display.toggle.coercions",
+				"when": "editorTextFocus && editorLangId == coq"
+			},
+			{
+				"key": "shift+alt+c",
+				"mac": "shift+ctrl+c",
+				"command": "extension.coq.display.toggle.coercions",
+				"when": "resourceScheme==coq-view"
+			},
+			{
+				"key": "shift+alt+n",
+				"mac": "shift+ctrl+n",
+				"command": "extension.coq.display.toggle.notations",
+				"when": "editorTextFocus && editorLangId == coq"
+			},
+			{
+				"key": "shift+alt+n",
+				"mac": "shift+ctrl+n",
+				"command": "extension.coq.display.toggle.notations",
+				"when": "resourceScheme==coq-view"
+			}
+		],
+		"menus": {
+			"editor/context": [
+				{
+					"when": "resourceLangId == coq",
+					"command": "extension.coq.interpretToPoint",
+					"group": "navigation"
+				},
+				{
+					"when": "resourceLangId == coq",
+					"command": "extension.coq.query.check",
+					"group": "queries"
+				},
+				{
+					"when": "resourceLangId == coq",
+					"command": "extension.coq.query.print",
+					"group": "queries"
+				},
+				{
+					"when": "resourceLangId == coq",
+					"command": "extension.coq.query.searchAbout",
+					"group": "queries"
+				},
+				{
+					"when": "resourceLangId == coq",
+					"command": "extension.coq.query.locate",
+					"group": "queries"
+				}
+			]
+		},
+		"snippets": [
+			{
+				"language": "coq",
+				"path": "./snippets/coq.json"
+			}
+		]
+	},
+	"scripts": {
+		"vscode:prepublish": "npm run cp-coqtopw && npm run compile-server && npm run compile-html-view && npm run compile",
+		"cp-coqtopw": "copyfiles -f ../coqtopw/bin/Release/coqtopw.exe ./",
+		"compile-server": "cd ../server && npm run compile && cd ../client",
+		"compile-html-view": "cd ../html_views && npm run compile && cd ../client",
+		"compile": "tsc -p ./",
+		"watch": "tsc -watch -p ./",
+		"postinstall": "node ./node_modules/vscode/bin/install"
+	},
+	"devDependencies": {
+		"typescript": "^2.2.1",
+		"vscode": "^1.0.3",
+		"copyfiles": "1.2.0",
+		"@types/node": "^7.0.5"
+	},
+	"dependencies": {
+		"vscode-jsonrpc": "3.1.0",
+		"vscode-languageclient": "3.1.0",
+		"xml2js": "^0.4.17",
+		"ws": "^3.3.1",
+		"opener": "^1.4.3",
+		"regenerate": "^1.3.2",
+		"unicode-9.0.0": "0.7.0",
+		"string-argv": "0.0.2",
+		"glob": "7.1.1",
+		"mustache": "3.0.1",
+		"@types/glob": "5.0.30",
+		"@types/xml2js": "^0.0.32",
+		"@types/chalk": "0.4.31",
+		"@types/ws": "^0.0.38",
+		"@types/mustache": "0.8.32"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/siegebell/vscoq.git"
```

Comment:
Is this correct?

Reply:
This is not changed by this PR.

Reply:
I take it as a no :-)

My eye just ended up seeing it, fix it when it fits, but before publishing since iirc it is displayed by the marketplace ui

Reply:
Oh yes, sure, I need to update a lot of things before I release to the marketplace.

Reply:
And yes, it's not correct :)